docs/core/actions.rst

```diff
@@ -137,32 +137,55 @@ events in :ref:`Events <events>`.
+| ``action_restart``                | Reset the whole conversation, usually triggered|
+|                                   | by using ``/restart``.                         |
++-----------------------------------+------------------------------------------------+
+| ``action_default_fallback``       | Undo the last user message (as if the user did |
+|                                   | not send it and the bot did not react) and     |
+|                                   | utter a message that the bot did not           |
+|                                   | understand. See :ref:`fallback-actions`.       |
++-----------------------------------+------------------------------------------------+
+| ``action_deactivate_form``        | Deactivate the active form and reset the       |
+|                                   | requested slot.                                |
+|                                   | See also :ref:`_section_unhappy`.              |
++-----------------------------------+------------------------------------------------+
+| ``action_revert_fallback_events`` | Revert events that occurred during the         |
+|                                   | TwoStageFallbackPolicy.                        |
+|                                   | See :ref:`fallback-actions`.                   |
++-----------------------------------+------------------------------------------------+
+| ``action_default_ask_affirmation``| Ask the user to affirm their intent.           |
+|                                   | It is suggested to overwrite this default      |
+|                                   | action with a custom action to have more       |
+|                                   | meaningful prompts.                            |
++-----------------------------------+------------------------------------------------+
+| ``action_default_ask_rephrase``   | Ask the user to rephrase their intent.         |                                   | did not send it) and utters a message that the |
++-----------------------------------+------------------------------------------------+
+| ``action_back``                   | Undo the last user message (as if the user did |
+|                                   | not send it and the bot did not react).        |
+|                                   | Can be triggered by sending ``/back``.         |
++-----------------------------------+------------------------------------------------+
 
 All the default actions can be overwritten. To do so, add the action name
 to the list of actions in your domain:
 
 .. code-block:: yaml
 
   actions:
-  - action_listen
+  - action_default_ask_affirmation
 
 Rasa Core will then call your action endpoint and treat it as every other
 custom action.
 
+The actions that can be triggered by the user (i.e. ``action_restart`` and
+``action_back``) are predicted by the MappingPolicy and will not be triggered if it
+is removed from the configuration file.
```

Comment:
Can you actually put this right info the table so that people definitely see it if they look up restart? I.e. `Reset the whole conversation, usually triggered by using ``/restart``. Requires MappingPolicy to be in the configuration.` or something like that

Reply:
Yes sounds good.
Is there a term for the core config analogous to "pipeline" for NLU? Saying the policy should be included in the configuration file seems a bit clumsy.

Reply:
I would just say policy configuration

Reply:
:+1: